query_plan.go

```diff
@@ -79,6 +82,58 @@ func (n *Node) Render() string {
 	return "\n" + tree.String()
 }
 
+type RenderedTreeWithStats struct {
+	Text         string
+	RowsTotal    string
+	Execution    string
+	LatencyTotal string
+}
+
+func (n *Node) RenderTreeWithStats() []RenderedTreeWithStats {
+	tree := treeprint.New()
+	renderTreeWithStats(tree, "", n)
+	var result []RenderedTreeWithStats
+	for _, line := range strings.Split(tree.String(), "\n") {
+		if line == "" {
+			continue
+		}
+
+		split := strings.Split(line, "\t")
```

Comment:
Probably it's safer to use [strings.SpritN](https://golang.org/pkg/strings/#SplitN) with N=2 in case that encoded JSON string has `\t`.

Reply:
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/66/commits/5158aff6414dc833c853fefd798c0f7dc9dfeabf